release.nix

```diff
@@ -97,6 +97,10 @@ let
 
         enableParallelBuilding = true;
 
+        __sandboxProfile = lib.sandbox.allowFileRead [
```

Comment:
Please add these lib functions to nixpkgs (master and cherry-picked back to the latest release)